pkg/sdn/plugin/sdn_controller.go

```diff
@@ -87,9 +87,11 @@ func (plugin *OsdnNode) alreadySetUp(localSubnetGatewayCIDR, clusterNetworkCIDR
 	}
 	found = false
 	for _, route := range routes {
-		if strings.Contains(route, clusterNetworkCIDR+" ") {
-			found = true
-			break
+		for _, clusterCIDR := range clusterNetworkCIDR {
+			if strings.Contains(route, clusterCIDR+" ") {
+				found = true
+				break
```

Comment:
We also want to break from routes for loop not just the clusterNetworkCIDR loop?